### DISCUSSION TITLE
예외처리

### DISCUSSION THREAD

**Discussion by klkim1913** [team-saviors/saview#107] [General]
### Description
예외처리 상황을 댓글로 추가해주세요!

### Todo
- [ ] 오류처리 세분화(jwt invalid로 다 걸러져버림)

### Due Date
- 09/2-

**Comment by Inhoob** [comment]
중복 닉네임, 중복 이메일

**Reply by klkim1913** [reply]
처리완료

**Comment by klkim1913** [comment]
탈퇴한 유저일 경우

**Reply by klkim1913** [reply]
처리완료
